lib/generator.rb

```diff
@@ -0,0 +1,43 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'erb'
+
+class Generator
```

Comment:
Should we rename this to DataGenerator as well to match the java program?

Reply:
It's not just a DocumentGenerator, but I guess Data is fine ... :)